.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,48 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8 black mypy
+    - name: Lint with flake8
+      run: |
+        flake8 *.py
+    - name: Lint with black
+      run: |
+        black --check --diff *.py
+    - name: Lint with mypy
+      run: |
+        mypy --strict *.py
```

Comment:
We may want to specify `.` instead of `*.py` in case we ever decide to restructure the repository to move source files into folders.

Reply:
Fair point, but also feels super unlikely for this wee library :) I'd prefer to leave as-is unless we think there will... ever... be more development here